frontend/src/pages/Player/Toolbar/NewCommentForm/CommentTextBody/CommentTextBody.tsx

```diff
@@ -46,6 +46,14 @@ const CommentTextBody = ({
 	}>()
 	const slackUrl = getSlackUrl(project_id ?? '')
 	const { slackLoading, syncSlack } = useSlackSync()
+	const [slackSynced, setSlackSynced] = useState(false)
+
+	const handleMentionsInputFocus = () => {
+		if (!slackSynced) {
```

Comment:
This syncs slack when the input is rendered, but will not re-sync if the user clicks a button or resets focus on the input